### ISSUE TITLE
Triple quotes around strings

### ISSUE THREAD

**Issue by jlandahl** [agourlay/json-2-csv#2]
Strings in the CSV output produced by `Json2Csv.convert` are surrounded by triple quotes, which is not standard and not understood by CSV readers I'm using to read the output (e.g. Spark's CSV reader). The result is that all strings end up with the triple quotes included in the values when read.

Here's a simple demonstration:

```scala
object Foo extends App {
  import com.github.agourlay.json2Csv.Json2Csv
  import java.io.ByteArrayOutputStream

  val json = """{"a": "abcdefg", "b": "hijklmnop"}"""
  val baos = new ByteArrayOutputStream
  Json2Csv.convert(Stream(json), baos)
  println(baos.toString("UTF8"))
}
```

The output is:

```
a,b
"""abcdefg""","""hijklmnop"""
```

**Comment by agourlay** [comment]
Thank you for opening this issue, this is indeed something that should be improved.

I just wrote a fix and released version ```0.3.0```, give it a try and tell me how it goes 😃

**Comment by agourlay** [comment]
closing due to inactivity
